### PR TITLE
Added bash comments ES translation

### PR DESCRIPTION
**Pull request by ezerw**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [x] 🚩 Other

## Description
Spanish Translation of "Bash comments".

## Related Tickets & Documents
No related ticket or document. Spanish Translation of [ebook/en/006-bash-comments.md](https://github.com/bobbyiliev/introduction-to-bash-scripting/blob/main/ebook/en/content/006-bash-comments.md)

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

**Comment by marcelozarate** [comment]
Spanish translation so far uses formal tone (usted). Can you please adapt it?

File: ebook/es/content/006-bash-comments.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Comentarios en Bash
+
+Como con cualquier otro lenguaje de programación, puedes agregar comentarios a tu script. Los comentarios son usados para dejar notas personales en el código.
```
Comment (marcelozarate) [inline ebook/es/content/006-bash-comments.md:3]:
```suggestion
Como con cualquier otro lenguaje de programación, puede agregar comentarios a su script. Los comentarios son usados para dejar notas personales en el código.
```

File: ebook/es/content/006-bash-comments.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Para hacerlo en Bash, necesitas agregar el simbolo `#` al comienzo de la línea. Los comentarios nunca aparecerán en la pantalla.
```
Comment (marcelozarate) [inline ebook/es/content/006-bash-comments.md:5]:
```suggestion
Para hacerlo en Bash, necesita agregar el simbolo `#` al comienzo de la línea. Los comentarios nunca aparecerán en la pantalla.
```

File: ebook/es/content/006-bash-comments.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# Este es un comentario y no aparecerá en la pantalla
+```
+
+Ahora agreguemos algunos comentarios a nuestro script:
+
+```bash
+#!/bin/bash
+
+# Pregúntale al usuario su nombre
+read -p "¿Cuál es tu nombre? " name
```
Comment (marcelozarate) [inline ebook/es/content/006-bash-comments.md:19]:
```suggestion
read -p "¿Cuál es tu nombre? " nombre
```

The example so far in the book uses the variable in its spanish version.

File: ebook/es/content/006-bash-comments.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+# Saluda al usuario
+echo "Hola $name"
+echo "Bienvenido a DevDojo!"
```
Comment (marcelozarate) [inline ebook/es/content/006-bash-comments.md:23]:
```suggestion
echo "¡Bienvenido a DevDojo!"
```

File: ebook/es/content/006-bash-comments.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+```
+
+Los comentarios son una excelente manera de describir algunas de las funciones más complejas directamente en tus scripts para que otras personas puedan orientarse en tu código con facilidad.
```
Comment (marcelozarate) [inline ebook/es/content/006-bash-comments.md:26]:
```suggestion
Los comentarios son una excelente manera de describir algunas de las funciones más complejas directamente en tus scripts para que otras personas puedan orientarse en su código con facilidad.
```

**Review by marcelozarate — CHANGES_REQUESTED** [review]
I left you some suggestions, hope you don't mind 👌

**Comment by ezerw** [comment]
Sure, thanks for the suggestions I'll do 👍

**Comment by ezerw** [comment]
Done, please let me know if I miss something.

File: ebook/es/content/006-bash-comments.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+```bash
+# Este es un comentario y no aparecerá en la pantalla
+```
+
+Ahora agreguemos algunos comentarios a nuestro script:
+
+```bash
+#!/bin/bash
+
+# Pregúntale al usuario su nombre
```
Comment (marcelozarate) [inline ebook/es/content/006-bash-comments.md:18]:
```suggestion
# Pregúntele al usuario su nombre
```

**Comment by ezerw** [comment]
Good catch, that should be it then!

**Review by bobbyiliev — APPROVED** [review]
Thank you! This is just amazing!

**Pull request merged**